test/framework/go.mod

```diff
@@ -0,0 +1,19 @@
+module sigs.k8s.io/cluster-api/test/framework
```

Comment:
This was actually removed in #2512 and it's going to ship with CAPI

Reply:
Thank you for pointing that out @vincepri ! 
But is there a good way to solve this problem? 😕 
```
ensure modules: /home/prow/go/src/sigs.k8s.io/cluster-api/cmd/clusterctl/test/e2e
go: sigs.k8s.io/cluster-api/test/framework@v0.0.0-20200125173702-54f26d7fd2b5: parsing ../../../../test/framework/go.mod: open /home/prow/go/src/sigs.k8s.io/cluster-api/test/framework/go.mod: no such file or directory 
```